templates/bake/Template/index.twig

```diff
@@ -51,6 +51,8 @@
 {% set columnData = Bake.columnData(field, schema) %}
 {% if columnData.type not in ['integer', 'float', 'decimal', 'biginteger', 'smallinteger', 'tinyinteger'] %}
                     <td><?= h(${{ singularVar }}->{{ field }}) ?></td>
+{% elseif columnData.null %}
```

Comment:
We could probably skip this check to make the code simpler.

Reply:
I thought its more desirable to generate cleaner code so not all fields will get the long version
```
<td><?= ${{ singularVar }}->{{ field }} === null ? '' : $this->Number->format(${{ singularVar }}->{{ field }}) ?></td>
```

Reply:
Sure, I guess its fine if our code has the check.